test/parallel/test-http2-compat-serverrequest-settimeout.js

```diff
@@ -12,6 +12,10 @@ server.on('request', (req, res) => {
   req.setTimeout(msecs, common.mustCall(() => {
     res.end();
   }));
+
+  res.on('timeout', common.mustCall());
+  req.on('timeout', common.mustCall());
```

Comment:
Can you please add some assertions that verifies that `'timeout'`  is not emitted?